wagtail/admin/templates/wagtailadmin/pages/confirm_delete.html

```diff
@@ -68,6 +68,20 @@
 
         <form action="{% url 'wagtailadmin_pages:delete' page.id %}" method="POST">
             {% csrf_token %}
+            {% if confirm_before_delete %}
+                <p class="status-msg warning">
+                    {% blocktrans with total_pages=descendant_count|add:1 %}
+                        This action will delete total <b>{{total_pages}}</b> pages.
+                    {% endblocktrans %}
+                </p>
+                <p>
```

Comment:
A bit painful sorry but we really should be using a label here instead of a `p` tag and associating it with the field.

```
<label class="w-mb-4" for="some-id">Please type <b>{{wagtail_site_name}}</b> to confirm.</label>
```